src/main/java/com/contentful/java/cma/ModuleEntries.java

```diff
@@ -227,6 +228,46 @@ public CMAEntry update(CMAEntry entry) {
   }
 
   /**
+   * Fetch all snapshots of an entry.
+   *
+   * @param entry the entry whose snapshots to be returned.
+   * @return an array of snapshots.
+   * @throws IllegalArgumentException if entry is null.
+   * @throws IllegalArgumentException if entry 's id is null.
+   * @throws IllegalArgumentException if entry 's space id is null.
+   */
+  public CMAArray<CMASnapshot> fetchAllSnapshots(CMAEntry entry) {
+    assertNotNull(entry, "entry");
+
+    final String entryId = getResourceIdOrThrow(entry, "entry");
+    final String spaceId = getSpaceIdOrThrow(entry, "entry");
+
+    return service.fetchAllSnapshots(spaceId, entryId).toBlocking().first();
+  }
+
+  /**
+   * Fetch a specific snapshot of an entry.
+   *
+   * @param entry      the entry whose snapshot to be returned.
```

Comment:
That is to align the description of the `entry` param with the description of the `snapshotId`